web_search_with_and/static/src/js/inherit_search_view.js

```diff
@@ -0,0 +1,53 @@
+odoo.define('horanet_web_search_with_and', function (require) {
```

Comment:
You are absolutly right, I migrated this module some time ago (for my company) and today decide to make a PR for OCA, it's an omission form our custom module